addon/components/tf-form/component.js

```diff
@@ -0,0 +1,30 @@
+/**
+  @module ember-ticketfly-fields
+ */
+import Ember from 'ember';
+import layout from './template';
+const { Component, guidFor, computed } = Ember;
+
+/**
+  @public
+  @class TfForm
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+  
+  tagName: 'form',
+  classNames: ['tf-form'],
+
+  inputGuid: computed({
+    get() {
+      return `id-${guidFor(this)}`;
+    }
+  }),
+
+  describedByGuid: computed({
+    get() {
+      return `described-by-${guidFor(this)}`;
+    }
+  })
+});
```

Comment:
`div` instead of `p` might be the way to go. I was thinking `<p>` was more semantically meaningful, but this [issue on the HTML spec](https://github.com/w3c/html/issues/558) seems to suggest that `<div>` is actual the safer accessibility choice.

Although I do still like `tf-form-field`, because you can also have a _group_ of fields -- which makes a bit more sense than a group of groups 😄.